super_mario_world_randomizer_porygone/manifest.json

```
@@ -2,9 +2,9 @@
     "name": "Super Mario World - Archipelago Tracker",
     "game_name": "Super Mario World",
     "platform": "Super Nintendo Entertainment System",
-    "package_version": "1.0.1",
+    "package_version": "2.0.0",
```

```suggestion
    "package_version": "2.0.0b1",
```
I prefer to mark pre-release versions as incremental betas until the AP side gets merged and things stabilize.

---

Ah, alright.